src/coreComponents/mesh/FaceElementSubRegion.cpp

```diff
@@ -1048,4 +1048,104 @@ std::set< std::set< globalIndex > > FaceElementSubRegion::getCollocatedNodes() c
   return result;
 }
 
+void FaceElementSubRegion::flipFaceMap( FaceManager & faceManager,
+                                        ElementRegionManager const & elemManager )
```

Comment:
Did you see the `fixNeighborMappingsInconsistency` function in the same file.
Does it conflict? Could the two fixes be done in one unique location?

Reply:
Honestly I did look into it but a lot is already happening n that function. I thought it was kind of easier to just add this cleanup step separately.